libs/fluid-elements/tab-group/src/lib/tab/tab.scss

```
@@ -0,0 +1,81 @@
+@import '~@dynatrace/fluid-design-tokens/global/spacing';
+@import '~@dynatrace/fluid-design-tokens/global/typography';
+
+:host {
+  /**
+   * Legibility definitions should probably be
+   * shipped or imported from a core
+  */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-rendering: optimizeLegibility;
+  margin-right: $fluid-spacing--medium;
+}
+
+/**
+* Disabled state
+*/
+:host([disabled]) {
+  pointer-events: none;
+
+  &:host .fluid-tab {
```

Should be fine as well.
```suggestion
  .fluid-tab {
```